sampleConfigurations/Templates/Introduction/C#/README.md

```diff
@@ -1,27 +1,30 @@
 ## Understanding WinGet Configuration Files
-This folder contains a WinGet Configuration File (*configuration.dsc.yaml*) that will work with the Windows Package Manager command line interface (`winget configure --file [path: configuration.dsc.yaml]`) or can be run using Microsoft Dev Home Device Configuration.
+This folder contains a [Windows Package Manager](https://learn.microsoft.com/en-us/windows/package-manager/winget/) (WinGet) [Configuration File](https://learn.microsoft.com/en-us/windows/package-manager/configuration/) (*configuration.dsc.yaml*) that will work with the WinGet command line interface (`winget configure --file [path: configuration.dsc.yaml]`) or can be run using [Microsoft Dev Home](https://learn.microsoft.com/en-us/windows/dev-home/) Device Configuration.
 
 When run, the `configuration.dsc.yaml` file will install the following list of applications:
 * Microsoft Visual Studio Community 2022
     * Required Visual Studio Workloads (ManagedDesktop, Universal)
 * GitHub Desktop
 
-
 The `configuration.dsc.yaml` file will also enable [Developer Mode](https://learn.microsoft.com/en-us/windows/apps/get-started/developer-mode-features-and-debugging) on your device. 
 
 ## How to use the WinGet Configuration File
+The following two options are available for running a WinGet Configuration file on your device. 
 
-### Windows Package Manager
+### 1. Windows Package Manager
 1. Download the `configuration.dsc.yaml` file to your computer.
 1. Open your Windows Start Menu, search and launch "*Windows Terminal*".
 1. Type the following: `CD [C:\Users\User\Download]`
```

Comment:
In the top-level README under sampleConfigurations, we use <> to indicate a path/variable that needs to be changed. In the rest of these files, we use []. I think I see <> much more often. We should probably consolidate.